index.html

```diff
@@ -107,5 +107,25 @@ <h4>Personal portfolio project</h4>
             </div>
         </div>
     </section>
+    <section id="contact">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/myyobwbp" method="POST">
+          <input type="text" id="client-name" name="name" placeholder="Enter your name" maxlength="30" required><br>
+          <input type="email" id="client-email" name="email" placeholder="Enter your email id" required><br>
+          <textarea id="client-message" name="message" placeholder="Write your message here" maxlength="500" required></textarea>        
+          <input id="submit-button" type="submit" value="Send" class="button">
+        </form>
+    </section>
+        <hr> 
+        <footer>
+            <img src="Icons/git.svg" alt="Github">
+            <img src="Icons/linkedin.svg" alt="LinkedIn">
+            <img src="Icons/Enabled.svg" alt="2">
+            <img src="Icons/twitter.svg" alt="Twitter">
+            <img src="Icons/Disabled.svg" alt="1">
+            <div>tetteyfioelisha@gmail.com</div>
```

Comment:
### **OPTIONAL**
- [ ] Pleases wrap your social app with a `ul` and  display each in a `li` with an `a tag `